packages/orbit-components/src/Dialog/index.tsx

```diff
@@ -150,51 +67,74 @@ const Dialog = ({
     }
   };
 
-  React.useEffect(() => {
-    const handleKeyDown = ev => {
-      if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
-        onClose();
-      }
-    };
-    window.addEventListener("keydown", handleKeyDown);
-    return () => window.removeEventListener("keydown", handleKeyDown);
-  }, [onClose]);
+  const handleKeyDown = (ev: React.KeyboardEvent<HTMLDivElement>) => {
+    if (ev.keyCode === KEY_CODE_MAP.ESC && onClose) {
+      onClose();
+    }
+  };
 
   const dialogID = useRandomId();
+  const descriptionId = useRandomId();
 
   const dialog = (
-    <StyledDialog
-      ref={wrapperRef}
-      data-test={dataTest}
-      id={id}
-      onClick={handleClose}
-      tabIndex={0}
+    <div
       role="dialog"
       aria-modal="true"
       aria-labelledby={dialogID}
+      aria-describedby={descriptionId}
     >
-      <StyledDialogCenterWrapper>
-        <StyledDialogContent shown={shown} ref={ref} id={dialogID} $maxWidth={maxWidth}>
-          {illustration && <IllustrationContainer>{illustration}</IllustrationContainer>}
-          <Stack spacing="XSmall" spaceAfter="medium">
-            {title && (
-              <Heading type="title3" align="center" largeMobile={{ align: "start" }}>
-                {title}
-              </Heading>
-            )}
-            {description && <Text type="secondary">{description}</Text>}
-          </Stack>
-          <Stack
-            direction="column-reverse"
-            spacing="XSmall"
-            largeMobile={{ direction: "row", justify: "end" }}
+      <div
+        ref={wrapperRef}
+        data-test={dataTest}
+        onKeyDown={handleKeyDown}
+        id={id}
+        onClick={handleClose}
+        tabIndex={0}
+        role="button"
+        className={cx([
+          "font-base",
+          "h-full w-full",
+          "p-md z-overlay bg-dialog-overlay-background box-border overflow-x-hidden",
```

Comment:
This bg now does not exist :sweat: You ended up changing but also removing the token?

Reply:
I removed the token, yes, now changed to the normal bg-static value 👍🏻 Thanks